### PR TITLE
Including the body of the message when forwarding emails from Mail client. 

### LINKED ISSUE #161: Bug: forwarding emails from Mail client doesn't include the body

### PR DESCRIPTION
**Pull request by varuntree**
As there is no api yet from the google's side to forward a message, we created our own method.
Tried, not to change and add to much of the complexity to the existing code.

for_image_test

Also added the Forwaded meta text for mail body.

Resolves #161

## Summary by CodeRabbit

- **New Features**
  - Enhanced email composition with new fields `messageText` and `messageHtml` for replies and forwards.

- **Refactor**
  - Improved code modularity and readability in email composition by adding `prepareReplyingToEmail` and `prepareForwardingEmail` functions.

- **Style**
  - Minor CSS adjustments for better UI consistency in email composition form.

### CODE REVIEW

File: apps/web/components/email-list/EmailPanel.tsx [hunk 1]
```diff
@@ -125,6 +126,45 @@ function EmailMessage(props: {
...
+    subject: `Re: ${message.headers.subject}`,
+    headerMessageId: message.headers["message-id"]!,
+    threadId: message.threadId!,
+    cc: message.headers.cc,
+    references: message.headers.references,
+    messageText: "",
+    messageHtml: "",
+  });
+
+  const prepareForwardingEmail = (message: ParsedMessage) => ({
```
Comment (elie222) [inline apps/web/components/email-list/EmailPanel.tsx:140]: Hey, wondering if we need the same code on the backend when we forward an email?

As there are emails that are forwarded solely via the backend so the logic would be the same thing

Comment (varuntree) [reply apps/web/components/email-list/EmailPanel.tsx:140]: Yeah, we can use the same code, or maybe it even makes more sense if we just use it only in backend. 
But, i can't find where exactly we forward the mails solely via backend, can you refer me there?

Comment (elie222) [reply apps/web/components/email-list/EmailPanel.tsx:140]: Sure. I'll take a look and share

**Comment by elie222** [comment]
@varuntree could you sign the CLA please (in the checks)

**Comment by varuntree** [comment]
@elie222 Hey, i did signed already, but for some reason its not updated, check all the sources.
image

**Comment by elie222** [comment]
Merged.

The code where we had something similar already is here:
`apps/web/utils/gmail/mail.ts`:

```ts
  const raw = await createRawMailMessage({
    to: options.to,
    cc: options.cc,
    bcc: options.bcc,
    subject: `Fwd: ${message.headers.subject}`,
    messageText: `${options.content ?? ""}
        
---------- Forwarded message ----------
From: ${message.headers.from}
Date: ${message.headers.date}
Subject: ${message.headers.subject}
To: <${message.headers.to}>

${message.textPlain}`,
    messageHtml: `<div>${options.content ?? ""}</div>

<div>---------- Forwarded message ----------</div>
<div>From: ${message.headers.from}</div>
<div>Date: ${message.headers.date}</div>
<div>Subject: ${message.headers.subject}</div>
<div>To: <${message.headers.to}></div>

<br>
```

Would be good to share the common code between these files.

**Pull request merged**